### PR TITLE
Avelino/database scheme via envvar

### LINKED ISSUE #679: database and scheme this in hardcode
It will not always be the `prest` database and `public` scheme, this name **must be dynamic**

in the first moment we should use an environment variable, in the future we will make the navigation fluid between all Postgres objects

### PR DESCRIPTION
**Pull request by avelino**
fixed: [#679](https://github.com/prest/prest/issues/679)

### CODE REVIEW

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 import {
-  withStyles,
-  WithStyles,
   createStyles,
   Theme,
+  withStyles,
+  WithStyles,
 } from "@material-ui/core/styles";
+import Toolbar from "@material-ui/core/Toolbar";
+import React from "react";
```
Comment (guidiego) [inline src/components/Layout.tsx:9]: Let's keep it consistent: 

```typescript
import React from "react"
import Any from "outsidelib"
import { named } from "outsidelib"

import AnyPersonal from "./my/path"
import { named 2 } from "./my/path
```

- Default imports from outside libraries
- Named imports from outside libraries
- Default imports from local project
- Named imports from local project

File: src/pages/[entity]/index.tsx [hunk 2]
```diff
@@ -1,24 +1,21 @@
-import React, { useState } from "react";
-
+import { Badge } from "@material-ui/core";
```
Comment (guidiego) [inline src/pages/[entity]/index.tsx:1]: Let's avoid this! Always get the component from `/Component` in material UI

```typescript
import Badge from "@material-ui/core/Badge";
```

Comment (guidiego) [reply src/pages/[entity]/index.tsx:1]: Same comment from previous: organize the imports

File: src/pages/[entity]/index.tsx [hunk 3]
```diff
@@ -1,24 +1,21 @@
...
-import { useRouter } from "next/router";
+import { PRestQuery, PRestTableShowItem } from "@postgresrest/node";
 import { GetServerSideProps } from "next";
-import { PRestTableShowItem } from "@postgresrest/node";
-import { DataGrid, GridColDef } from "@material-ui/data-grid";
-import { makeStyles } from "@material-ui/core/styles";
-import { PRestQuery } from "@postgresrest/node";
-import { Badge } from "@material-ui/core";
+import { useRouter } from "next/router";
+import React, { useState } from "react";
```
Comment (guidiego) [inline src/pages/[entity]/index.tsx:10]: React should always be the first import

**Review by guidiego — APPROVED** [review]
LGTM! Only please fix the import orders and take a second look at how you are importing some of the Material UI components! Once done it could merge!

File: src/pages/[entity]/index.tsx (hunk 3 above)
Comment (avelino) [reply src/pages/[entity]/index.tsx:10]: why should react be the first import?

my editor is set to orderna imports

File: src/pages/[entity]/index.tsx (hunk 2 above)
Comment (avelino) [reply src/pages/[entity]/index.tsx:1]: by doing the import without  `{}`:
```ts
import Badge from "@material-ui/core/Badge";
```

gave error when using the element:
```
import Badge
JSX element type 'Badge' does not have any construct or call signatures.ts(2604)
```

**Pull request merged**